src/Compilers/CSharp/Portable/Symbols/Synthesized/SynthesizedSimpleProgramEntryPointSymbol.cs

```diff
@@ -281,6 +281,31 @@ internal override bool IsDefinedInSourceTree(SyntaxTree tree, TextSpan? definedW
 
         public SyntaxNode ReturnTypeSyntax => CompilationUnit.Members.First(m => m.Kind() == SyntaxKind.GlobalStatement);
 
+        /// <summary>
+        /// Returns a bag of applied custom attributes and data decoded from well-known attributes. Returns null if there are no attributes applied on the symbol.
+        /// </summary>
+        /// <remarks>
+        /// Forces binding and decoding of attributes.
+        /// </remarks>
+        internal override CustomAttributesBag<CSharpAttributeData> GetAttributesBag()
```

Comment:
>override [](http://example.com/codeflow?start=17&length=8)

As I commented on initial declaration of this method, this change looks suspicious.